src/index.ts

```diff
@@ -29,22 +33,26 @@ export namespace httpclient {
 			}
 		}
 
+		// Takes a Request and returns the body of the method callForResponse with the same parameter
```

Comment:
It doesn't return the body of the method `callForResponse` method, it returns the body of the Response object returned by `callForResponse`